data.json

```
@@ -455,4 +455,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/22374829/104178614-a8acea80-5430-11eb-8ee8-1a7bb89248b2.png",
     "issueNumber": 237
   }
```

Please add a comma here
```suggestion
  },
```